rust/protocol/tests/groups.rs

```diff
@@ -86,10 +86,10 @@ fn group_using_context_arg() -> Result<(), SignalProtocolError> {
     async {
         let mut csprng = OsRng;
 
-        let sender_address = ProtocolAddress::new("+14159999111".to_owned(), 1);
+        let sender_address = ProtocolAddress::new("+14159999111".to_owned(), 1.into());
         let distribution_id = Uuid::from_u128(0xd1d1d1d1_7000_11eb_b32a_33b8a8a487a6);
 
-        let x = Box::new(1);
+        let x: Box<DeviceId> = Box::new(1.into());
```

Comment:
This isn't a device ID, just a placeholder to test context.

Reply:
Thanks--reverted!